hints/amigaos.sh

```diff
@@ -6,3 +6,5 @@ for f in amigaos4/*.h amigaos4/*.c
 do
   cp -f $f .
 done
+
+ccflags="$ccflags -fno-stack-protector"
```

Comment:
This looks outside of the scope of the current Pull Request, were you able to test this change?

Reply:
No

Reply:
If you prefer, #17857 is safer.